tests/osn-tests/src/test_osn_global.ts

```diff
@@ -16,44 +18,115 @@ describe('osn_global', () => {
         }
     });
 
+    // Shutdown OBS process
     after(function(done) {
-        this.timeout(3000);
+        this.timeout(5000);
         obs.shutdown();
         obs = null;
-        setTimeout(done, 2000);
+        setTimeout(done, 3000);
```

Comment:
I'm really really not a fan of all of these timeouts. I don't get why we need them.

Reply:
We should be listening to an event here, we should force that to be a synchronous operation.

Reply:
Created a promise with IPC.disconnect() and used await to wait it complete since disconnect is async. Removed all the setTimeouts and the tests run fine with the OBS process being created and closed properly. That is enough for this or I'm doing something wrong here?